generator/src/main/java/com/scottlogic/deg/generator/GenerateExecute.java

```diff
@@ -14,22 +16,33 @@
     private final GenerationEngine generationEngine;
     private final GenerationConfigSource configSource;
     private final OutputTarget fileOutputTarget;
+    private CommandLineValidator validator;
 
     @Inject
     public GenerateExecute(GenerationConfig config,
                            ProfileReader profileReader,
                            GenerationEngine generationEngine,
                            GenerationConfigSource configSource,
-                           OutputTarget fileOutputTarget) {
+                           OutputTarget fileOutputTarget,
+                           CommandLineValidator validator) {
         this.config = config;
         this.profileReader = profileReader;
         this.generationEngine = generationEngine;
         this.configSource = configSource;
         this.fileOutputTarget = fileOutputTarget;
+        this.validator = validator;
     }
 
     @Override
     public void run() {
+
+        ValidationResult validationResult = validator.validateCommandLine(config);
+
+        if (!validationResult.isValid()) {
+                validationResult.errorMessages.forEach(System.err::println);
```

Comment:
This will print 2 return lines (one from the end of the message and another at the end of the `println` call. If you remove the `\n` from the end of the message the problem should be solved.